test/integration/api.bats

```diff
@@ -9,7 +9,7 @@ function teardown() {
 
 @test "docker info should return the number of nodes" {
 	start_docker 3
-	start_manager
+	swarm_manage `join , ${HOSTS[@]}`
```

Comment:
@aluzzardi I refactor `swarm_manage` to make it reusable across tests.
Should I leave `join , ${HOSTS[@]}`? Or would you prefer something function `join_hosts`?